pkg/image/builder.go

```diff
@@ -91,7 +91,7 @@ func ProcessImages(srcRegistry, destRegistry registry.RegistryOptions, appSlug s
 	return newImages, nil
 }
 
-func GetPrivateImages(upstreamDir string, checkedImages map[string]ImageInfo, allPrivate bool, dockerHubRegistry registry.RegistryOptions, useHelmInstall map[string]bool) ([]string, []k8sdoc.K8sDoc, error) {
+func GetPrivateImages(upstreamDir string, checkedImages map[string]ImageInfo, allPrivate bool, dockerHubRegistry registry.RegistryOptions, parentHelmChartPath string, useHelmInstall map[string]bool) ([]string, []k8sdoc.K8sDoc, error) {
```

Comment:
this function is not just for helm charts though, a bit confusing that there is a parameter named `parentHelmChartPath`

Reply:
maybe pass base? or convert this function to be something like
```
func (b *Base) GetPrivateImages(
```

Reply:
`maybeParentHelmChartPath`?  it's actually a bit of an invariant.  without parent chart, it's empty string, which still works with the `useHelmInstall` map

Reply:
i don't have any bright ideas. no strong objection to keep it as is. maybe aggregate all of these params in a new options struct or something

Reply:
the name makes sense to me given existing context